.env.example

```diff
@@ -0,0 +1,10 @@
+REACT_APP_API_KEY={api_key}
+REACT_APP_AUTH_DOMAIN={auth_domain}
+REACT_APP_DATABASE_URL={db_url}
+REACT_APP_PROJECT_ID={project_id}
+REACT_APP_STORAGE_BUCKET={storage_bucket}
+REACT_APP_MESSAGING_SENDER_ID={messaging_sender_id}
+REACT_APP_APP_ID={app_id}
+REACT_APP_MEASUREMENT_ID={measurement_id}
+
+GENERATE_SOURCEMAP=false
```

Comment:
This line fixes the victory library console log spam by ignoring the problem for now 😏